modules/oe_whitelabel_starter_event/templates/node--oe-sc-event--teaser.html.twig

```diff
@@ -7,17 +7,19 @@
 {% set _title %}
   <a class="standalone" href="{{ url }}">{{ label }}</a>
 {% endset %}
-{% set _content %}
-  <span class="text-muted text-nowrap me-4-5">{{ content.oe_sc_event_dates }}</span>
-{% endset %}
 {% block content %}
 <article{{attributes}}>
   {{ pattern('card', {
     variant: 'search',
     title: _title,
     text: content.oe_summary,
-    image: image,
-    content: _content
+    image: image ? {
+      path: image.src,
+      alt: image.alt,
+    },
+    meta: [
+      content.oe_sc_event_dates,
```

Comment:
```suggestion
      content.oe_sc_event_location|field_value,
      content.oe_sc_event_dates|field_value
```

and use **inline address formatter** for the location

Reply:
in the end we wont do it because the address can be quite long, so we need to be able to configure this. Postponed.